src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx

```diff
@@ -3,16 +3,34 @@ import React from 'react';
 import {t} from 'app/locale';
 import styled from 'react-emotion';
 
+import {analytics} from 'app/utils/analytics';
 import AsyncComponent from 'app/components/asyncComponent';
 import Button from 'app/components/button';
 import {PanelItem} from 'app/components/panels';
 import {promptsUpdate} from 'app/actionCreators/prompts';
+import ProgressBar from 'app/views/projectReleases/progressBar';
 
 const STEPS = {
-  tag: t('Tag an error'),
-  repo: t('Link to a repo'),
-  commit: t('Associate commits'),
-  deploy: t('Tell sentry about a deploy'),
+  tag: {
+    desc: t('Tag an error'),
+    url: 'tag-errors',
+    msg: 'knowing which errors were introduced in a release, ',
```

Comment:
I can work on making this bold in particular spots to highlight bits of each but this works for now

Reply:
not too important